README.md

```diff
@@ -147,3 +147,21 @@ exit
 ```
 
 [SBT 1.x documentation](https://www.scala-sbt.org/1.x/docs/index.html)
+
+## IDE setup
+
+### IntelliJ IDEA 
+
+1. [Download IntelliJ (free Community edition is fine)](https://www.jetbrains.com/idea/download/#section=mac)
+
+2. Install and open IntelliJ
+
+3. Install Scala plugin: _Configure -> Plugins -> Browse Repositories -> Scala_
```

Comment:
I'm not sure about CE but normally intellij offers the chance to install plugins when you run it for the very first time.
Maybe add a simple note about it. If they don't see it the proceed to the manual installation.